### DISCUSSION TITLE
Command palette actions

### DISCUSSION THREAD

**Discussion by JeroenVdb** [Kong/insomnia#7030] [Ideas]
Now the previous event switcher is replaced by the more versatile Command Palette we can think about adding extra functionality besides switching between requests and collections.

I have a few things in mind that I would think are useful to have at hand in the command palette. I would like to present a few things I have in mind and would be interested to hear what other ideas are.

1. Switch environment: in a workspace you have a list of actions to switch to a different environment.
<img width="682" alt="Screenshot 2024-01-29 at 09 25 56" src="https://github.com/Kong/insomnia/assets/657797/6f9b4da0-f935-4e6b-a2e4-d0cc9411c55d">

2. Show a list of the last 10 requests across workspaces

For number 1 I'm fairly sure I can PR it to the project. Number 2 I'm would have to digg in further.
Any more ideas?

**Comment by gatzjames** [comment]
@JeroenVdb Switching environments would be cool!
For the list of requests we want to explore showing all the requests inside a collection/document for all organizations but it comes with some perf considerations for users with many requests/collections
